token-holders/staking-rules.md

```diff
@@ -0,0 +1,12 @@
+These are the current rules of Pyth staking:
+
+1. Pyth staking is organized in one-week epochs. Each Thursday at 00:00am UTC (midnight between Wednesday and Thursday), a new epoch starts. 
+2. When a user stakes their Pyth tokens, their stake will only become active at the start of the next epoch.   
+3. When a user requests to unlock their Pyth tokens, the tokens will go into cooldown at the end of the current epoch and will be withdrawable after one full epoch.
+
+Examples: 
+- Daniel requests to lock his tokens in epoch 1, his tokens will become locked on epoch 2.
+- Alice requests to unlock her tokens during epoch 1. Her tokens will become unlocked at the start of epoch 3 and only then will she be able to withdraw them.
```

Comment:
Alice requests to unlock her tokens during epoch 1. Her tokens remain locked until the end of epoch 1. Her tokens cool down for the entirety of epoch 2, and become unlocked at the start of epoch 3 and only then will she be able to withdraw them.